tests/unit/test_main.py

```diff
@@ -785,3 +785,132 @@ async def test_get_traces_any(self, mock_stored_flows, event_loop):
         assert result[0][0]["out"] == {"port": 3}
         assert result[1][0]["type"] == "incomplete"
         assert result[1][0]["out"] is None
+
+    def test_do_match(self):
+        """Test do_match."""
+
+        # Check dl_vlan
+        flow = {'flow': {'match': {'dl_vlan': 10}}}
```

Comment:
The test case looks good, but it's repeating many `args` `resp`, the idiomatic way with `pytest` when you need to test against multiple/different inputs is to use the `@pytest.mark.parametrize` decorator, and then parametrize `flow,args,should_match` (dict, dict, bool), take a look [on some unit tests examples from flow_manager for instance if you need a reference](https://github.com/kytos-ng/flow_manager/blob/master/tests/unit/test_match_13.py#L53-L73)